src/AirtableManager.php

```diff
@@ -119,7 +119,13 @@ protected function createAirtable($table)
             $httpLogFormat = null;
         }
 
-        $client = new AirtableApiClient($base, $table, $access_token, $httpLogFormat);
+        if ($this->app['config']['airtable.typecast']) {
```

Comment:
Is this conditional required, if we are going to default that config option to false?

Reply:
You are right, Pushed a cleanup commit.

Reply:
@swilla I would like to push this into production, Sorry to bother you again but can we have it merged?